### PR TITLE
Fix bug in menu selection for the local documentation.

### PR DESCRIPTION
**Pull request by fabienrohrer**
Fix issue #103

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

File: js/viewer.js [hunk 1]
```diff
@@ -389,7 +389,9 @@ function changeMenuSelection() {
         var href = a.getAttribute("href");
         var selection;
         if (local) {
-          if (href.indexOf("page=" + setup.page) > -1)
+          var pageIndex = href.indexOf("page=" + setup.page);
+          // Note: 5 matches with the "page=" string length
+          if (pageIndex > -1 && href[5 + pageIndex + setup.page.length] != "-")
```
Comment (stefaniapedrazzi) [inline js/viewer.js]: I'm not sure if it is enough to check the character "-".
Will this fix also work with a page file named "introductiontowebots.md" exists?

Comment (fabienrohrer) [inline js/viewer.js]: You're right, this can be problematic indeed.

Comment (fabienrohrer) [inline js/viewer.js]: The test is more generic now.

**Comment by fabienrohrer** [comment]
👍

**Comment by stefaniapedrazzi** [comment]
:+1:

**Pull request merged**